File: C_Programming/Practise_Programs/PP_048.c
/*
==============================================================================
 Program     : Display a String Five Times Using do-while Loop

 Description : Displays a string five times using a do-while loop.

 Author      : Shubham Gadhe
==============================================================================
*/

#include<stdio.h>

int main()
{
    int iCnt = 0;

    iCnt = 1;

    // Display the string five times.
    do
    {
        printf("Jay Ganesh...\n");
        iCnt++;
    }while(iCnt <= 5);
    
    return 0;
}

/*
==============================================================================
 Sample Output
------------------------------------------------------------------------------
 Jay Ganesh...
 Jay Ganesh...
 Jay Ganesh...
 Jay Ganesh...
 Jay Ganesh...
==============================================================================
*/

File: Java_Programming/Practise_Programs/PP_048.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_048
//
//  Description       : This program accepts a number from the user
//                      and displays lowercase alphabets starting
//                      from ASCII value 97 for the specified count.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 06/08/2026
//
//  Time Complexity   : O(n)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

import java.util.*;

class PP_048
{
    //////////////////////////////////////////////////////////////////
    //
    //  Function Name    : Display()
    //
    //  Description      : It is used to display lowercase
    //                     alphabets by initializing the
    //                     character variable with its
    //                     ASCII value (97).
    //
    //  Author           : Shubham Somanath Gadhe
    //  Date             : 06/08/2026
    //
    //////////////////////////////////////////////////////////////////

    public static void Display(int iNo)
    {
        int iCnt = 0;
        char ch = '\0';

        for(iCnt = 1, ch = 97; iCnt <= iNo; iCnt++, ch++)
        {
            System.out.print(ch + "\t");
        }

        System.out.println();
    }

    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the number of elements : ");
        iValue = sobj.nextInt();

        Display(iValue);
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input  :
//      5
//
//  Sample Output :
//      a    b    c    d    e
//
//////////////////////////////////////////////////////////////////